### DISCUSSION TITLE
Building myMPD on Fedora 36 fails

### DISCUSSION THREAD

**Discussion by avalx** [jcorporation/myMPD#885] [Q&A] [answered]
I'm trying to build a rpm package for the dev version but I got an error. When I run `build.sh pkgrpm` it immediately fails with error:

error: line 43: %package debuginfo
: package mympd-debuginfo already exists

My system is Fedora 36 with all the latest updates. Some time ago this process was working without problems, so I'm wondering what has changed to cause this error. Any ideas?

**Comment by jcorporation** [comment, marked as answer]
RPM builds fine here: https://build.opensuse.org/package/live_build_log/home:jcorporation/myMPD/Fedora_36/x86_64

It could be a problem with line 34: `%debug_package` in the file https://github.com/jcorporation/myMPD/blob/master/contrib/packaging/rpm/mympd.spec

Please remove it and try again.

**Reply by avalx** [reply]
That did the trick - now it builds fine. Will I have to do this for all future versions as well?

Thanks for the tip.

**Reply by jcorporation** [reply]
No, this will be fixed for the next release.
